src/CollAction/ValidationAttributes/FileSizeAttribute.cs

```diff
@@ -0,0 +1,26 @@
+﻿using Microsoft.AspNetCore.Http;
+using System.ComponentModel.DataAnnotations;
+
+namespace CollAction.ValidationAttributes
+{
+    public class FileSizeAttribute : ValidationAttribute
```

Comment:
This is used in EditProjectViewModel.cs line 67 and CreateProjectViewModel.cs line 65.